backend/fregepoc/indexers/tests/test_bitbucket.py

```diff
@@ -0,0 +1,80 @@
+import json
+from pathlib import Path
+
+import pytest
+import requests
+from django.conf import settings
+from pytest_mock import MockerFixture
+
+from fregepoc.indexers.models import BitbucketIndexer
+
+MOCK_PATH = settings.BASE_DIR / "fregepoc" / "indexers" / "tests" / "__mock__"
```

Comment:
I think you can move this constant variable to a separate file (maybe constants.py), I use this in my tests too.

Reply:
refactored in 909d8fd